src/main/java/com/gregtechceu/gtceu/core/mixins/ae2/PowerUnitsMixin.java

```diff
@@ -0,0 +1,50 @@
+package com.gregtechceu.gtceu.core.mixins.ae2;
+
+import appeng.api.config.PowerUnits;
+import com.llamalad7.mixinextras.injector.ModifyReturnValue;
+import org.spongepowered.asm.mixin.Debug;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Unique;
+import org.spongepowered.asm.mixin.gen.Invoker;
+import org.spongepowered.asm.mixin.injection.At;
+
+import java.util.Arrays;
+
+// Reasoning: The Power Unit enum is used to store
+// the different power units and their lang/conversion
+// factors in AE2. We want to add EU to this.
+@Debug(export = true)
+@Mixin(value = PowerUnits.class, remap = false)
+public class PowerUnitsMixin {
+
+    @Unique
+    private static PowerUnits gtceu$EU_UNIT;
+
+    @Invoker(value = "<init>")
+    private static PowerUnits gtceu$invokeConstructor(String internalName, int ordinal, String unlocalizedName,
+                                                      String textRepresentation) {
+        throw new AssertionError();
+    }
+
+    @ModifyReturnValue(method = "values", at = @At("RETURN"))
+    private static PowerUnits[] gtceu$addEUToValues(PowerUnits[] original) {
+        if (gtceu$EU_UNIT == null) {
+            gtceu$EU_UNIT = gtceu$invokeConstructor("EU", original.length, "gui.ae2.units.eu", "EU");
+            // Conversion ratio of EU to AE = EU to FE * FE to AE;
+            float FEtoAE = 0.5f;
+            float EUtoFE = 4f;
```

Comment:
These will need to be looked into, since in order to use the accurately configured ratios, we need to defer this somehow since this class is loaded by AE2's config class before GT's config is loaded